cl/search/tests.py

```diff
@@ -1048,6 +1050,22 @@ def _perform_wildcard_search(self):
         result_count = self.browser.find_element_by_id("result-count")
         self.assertIn("Opinions", result_count.text)
 
+    def test_main_query_cleanup(self):
+        # Dora goes to CL and performs a Search using a numbered citation
+        # (e.g. "12-9238" or "3:18-cv-2383")
+        self.browser.get(self.live_server_url)
+        searchbox = self.browser.find_element_by_id("id_q")
+        searchbox.clear()
+        searchbox.send_keys("19-2205")
+        searchbox.submit()
+        # without the cleanup_main_query function, there are 4 results
+        # with the query, there should be none
```

Comment:
I think I'd do two tests here and prove they return the same thing. One with quotes, one without?

Reply:
Not sure how to implement this one. In order to get the 4 results as if the quotes were not present, you'd need to disable the code in search_utils, since it's automatically encoded.

The issue is how to mock the fail -- maybe a unit test on the input / output alone (i.e. testing  the cleanup_main_query function)?